xml/Microsoft.Extensions.Primitives/CancellationChangeToken.xml

```diff
@@ -83,7 +83,7 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Indicates if this token will pro-actively raise callbacks. If <see langword="false" />, the token consumer must poll <see cref="P:Microsoft.Extensions.Primitives.IChangeToken.HasChanged" /> to detect changes.</summary>
         <value>To be added.</value>
```

Comment:
Can you make sure to manually add descriptions to fields that still say "To be added"?
Note: Remarks are optional.

Reply:
@maryamariyan and just to note, `To be added.` is a magic keyword to actually omit that description entirely from the documentation published. It will look really bad if you don't fill out that information.

Reply:
You can read the [wiki section for Property Values](https://github.com/dotnet/dotnet-api-docs/wiki/Property-Value) to learn how to properly word their description.